pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -226,6 +80,10 @@ def fractional_octave_bands(
         >>> plt.tight_layout()
 
     """
+    warnings.warn((
+            "This function will be deprecated in pyfar 0.6.0. "
```

Comment:
Can you correct this to 0.8.0. The new classes will be released in 0.6.0 and deprecated to minor releases later. (See also below)